environment/src/main/java/com/sequenceiq/environment/environment/service/DistroxService.java

```diff
@@ -0,0 +1,77 @@
+package com.sequenceiq.environment.environment.service;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.concurrent.TimeUnit;
+import java.util.stream.Collectors;
+
+import javax.ws.rs.ProcessingException;
+import javax.ws.rs.WebApplicationException;
+
+import org.jetbrains.annotations.NotNull;
```

Comment:
remove this please

Reply:
why did you add this to project than if it is not usable?

Reply:
javax.validation.constraints.NotNull should be used

Reply:
ok, but Denes added jetbrains dependency to project and Idea added this import automatically. So it does not disturb me, It is ok for me